src/main/cpp/t03_bigger.cpp

```diff
@@ -19,5 +19,18 @@
 using namespace std;
 
 int t03_bigger() {
-
+    long n = 0;
+    cin >> n;
+    const long kol = n;
+    long mas[kol];
+    for (int i = 0; i < kol; i++)
+    {
+        int znachenie = 0;
+        cin >> znachenie;
+        if (znachenie > mas[i-1] && i != 0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either the condition 'i!=0' is redundant or the array 'mas[0]' is accessed at index -1, which is out of bounds.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4478840)